android/build.gradle

```diff
@@ -30,12 +30,14 @@ repositories {
     }
     google()
     jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'
 
-    implementation 'com.thanosfisherman.wifiutils:wifiutils:1.6.1'
+
+    implementation 'com.github.eliaslecomte:WifiUtils:ffbdc86'
```

Comment:
I want to switch to WifiUtils as soon as they release a new version (code has been merged in master already).

Reply:
@eliaslecomte, should we wait until a new version of WifiUtils is released for merging this PR?

Reply:
I think that's best :-). Let's see if I can get in touch.